gcc/testsuite/rust/compile/torture/issue-1432.rs

```diff
@@ -67,6 +67,5 @@ impl_uint!(
     u16 = "u16",
     u32 = "u32",
     u64 = "u64",
-    u128 = "u128",
```

Comment:
Interesting that this causes failures on 32 bits. How do we store 128 bits values? I believe they should simply be stored as 4 words on the stack [godbolt link](https://godbolt.org/#g:!((g:!((g:!((h:codeEditor,i:(filename:'1',fontScale:14,fontUsePx:'0',j:1,lang:rust,selection:(endColumn:1,endLineNumber:11,positionColumn:1,positionLineNumber:11,selectionStartColumn:1,selectionStartLineNumber:11,startColumn:1,startLineNumber:11),source:'pub+fn+foo()+-%3E+u64+%7B%0A++++let+a+%3D+15%3B%0A++++let+b+%3D+15%3B%0A%0A++++a+%2B+b%0A%7D%0A%0Apub+fn+bar()+-%3E+u128+%7B%0A++++let+a+%3D+15%3B%0A++++let+b+%3D+15%3B%0A%0A++++a+%2B+b%0A%7D'),l:'5',n:'0',o:'Rust+source+%231',t:'0')),k:50,l:'4',n:'0',o:'',s:0,t:'0'),(g:!((h:compiler,i:(compiler:r1620,filters:(b:'0',binary:'1',commentOnly:'0',demangle:'0',directives:'0',execute:'1',intel:'0',libraryCode:'0',trim:'1'),flagsViewOpen:'1',fontScale:14,fontUsePx:'0',j:1,lang:rust,libs:!(),options:'-C+opt-level%3D0+--target+i686-unknown-linux-gnu',selection:(endColumn:1,endLineNumber:1,positionColumn:1,positionLineNumber:1,selectionStartColumn:1,selectionStartLineNumber:1,startColumn:1,startLineNumber:1),source:1,tree:'1'),l:'5',n:'0',o:'rustc+1.62.0+(Rust,+Editor+%231,+Compiler+%231)',t:'0')),k:50,l:'4',n:'0',o:'',s:0,t:'0')),l:'2',n:'0',o:'',t:'0')),version:4)